contrib/export/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec"
+	extypes "github.com/cosmos/cosmos-sdk/contrib/export/types"
+	"github.com/cosmos/cosmos-sdk/contrib/export/v036"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/tendermint/types"
+	"log"
+)
+
+var (
+	migrationMap = extypes.MigrationMap{
+		"v0.36": v036.Migrate,
+	}
+	//source        string
+	target        string
+	importGenesis string
+)
+
+const (
+	// FlagSource will support multiple version upgrades
+	FlagSource = "source"
+)
+
+func migrateGenesisCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "genesis [v0.34] [~/my/genesis.json]",
```

Comment:
```suggestion
		Use:   "genesis [version] [~/my/genesis.json]",
```

Reply:
I was thinking that putting an example makes it easier to understand in which format to write the version: 034, 0,34, v0.34...